### ISSUE TITLE
Widget screen: Custom block styles are not available when running WP 5.7

### ISSUE THREAD

**Issue by carolinan** [WordPress/gutenberg#25808]
**Describe the bug**
When a block with a custom block style (register_block_style) is added to a widget area,
the custom style is not present in the block inserter or block sidebar.

**To reproduce**
1. Activate Gutenberg
2. Activate a theme with custom block styles, for example Seedlet.
3. In the widgets screen, place a block that has a custom block style, for example, _latest posts_ in Seedlet.
4. When adding the block, see that the block style is not showing in the block inserter.
5. With the block selected, open the block sidebar, and view the styles section.
6. See that the custom styles are missing, or if it is the only style, that the styles section is missing.

**Expected behavior**
I expect all options to be available when placing the block.

**Editor version (please complete the following information):**
- WordPress version: 5.5.1
-Gutenberg version 9.1

**Comment by draganescu** [comment]
I was able to reproduce this on Gutenberg master as of Oct. 17th.

**Comment by talldan** [comment]
I can also replicate using seedlet. What's interesting is that from the console this still works:
```jsx
wp.blocks.registerBlockStyle( 'core/latest-posts', {
    name: 'fancy-latest-posts',
    label: 'Fancy Latest Posts',
} );
```

**Comment by noisysocks** [comment]
Thanks for the report! I can only reproduce it when running the Gutenberg plugin on WP 5.7. It's fixed in 5.8-alpha. Going to de-prioritse the issue in the Widgets project board.

**Comment by draganescu** [comment]
If it's fixed in 5.8-alpha why not close this issue?

**Comment by carolinan** [comment]
I can confirm that this is fixed. I don't believe there are any backporting to 5.7 concerns, as far as I know, that is not planned.
I am not sure why it is listed in _maybe later_.  Please reopen if needed.
